RecoVertex/PrimaryVertexProducer/interface/PrimaryVertexProducerAlgorithm.h

```diff
@@ -39,7 +39,6 @@
 #include "RecoVertex/PrimaryVertexProducer/interface/TrackFilterForPVFinding.h"
 #include "RecoVertex/PrimaryVertexProducer/interface/HITrackFilterForPVFinding.h"
 #include "RecoVertex/PrimaryVertexProducer/interface/GapClusterizerInZ.h"
-#include "RecoVertex/PrimaryVertexProducer/interface/DAClusterizerInZ.h"
 #include "RecoVertex/KalmanVertexFit/interface/KalmanVertexFitter.h"
 #include "RecoVertex/AdaptiveVertexFit/interface/AdaptiveVertexFitter.h"
 //#include "RecoVertex/VertexTools/interface/VertexDistanceXY.h"
```

Comment:
Maybe clean up this commented line while we're here.

Reply:
This PrimaryVertexProducerAlgorithm is obsolete and doesn't work anyway. The only reason why it is still here is, that it is referenced in other parts of CMSSW.  It will be removed in a separate pull request.